spec/index.bs

```diff
@@ -4411,11 +4418,11 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
                 :: If |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/type}} is:
                     <dl class=switch>
                         : {{GPUBufferBindingType/"uniform"}}
-                        :: The |binding| is a uniform buffer.
+                        :: |binding| is declared with the storage class `uniform`.
                         : {{GPUBufferBindingType/"storage"}}
-                        :: The |binding| is a storage buffer.
+                        :: |binding| is declared with the storage class `storage`.
```

Comment:
we should say that the access is read-write here

Reply:
Can't the access mode be anything in this case? (`read_write`, `read`, or `write`)

Reply:
Discussed and it's unclear whether that would be possible (e.g. `read` access may cause user or compiler to assume the underlying storage never changes); it should just be strict for now.